tests/lib/prepare.sh

```diff
@@ -432,6 +432,82 @@ EOF
     rm -rf "$UNPACK_DIR"
 }
 
+
+uc20_build_initramfs_kernel_snap() {
+    # carries ubuntu-core-initframfs
+    add-apt-repository ppa:snappy-dev/image -y
+    apt install ubuntu-core-initramfs -y
+
+    local TARGET="$1"
+    # kernel snap is huge, unpacking to current dir
+    unsquashfs -d repacked-kernel pc-kernel_*.snap
+
+    local SNAPD_UNPACK_DIR="/tmp/snapd-unpack"
+    dpkg-deb -x "$SPREAD_PATH"/../snapd_*.deb "$SNAPD_UNPACK_DIR"
```

Comment:
note that this is unnecessary, since at this point we have already installed the snapd deb from the PR, so we can just copy /usr/lib/snapd/snap-bootstrap into the container instead of extracting the deb

Reply:
actually I fixed this in a push since the tests need to be fixed anyways